give/src/components/Title/Title.js

```diff
@@ -0,0 +1,70 @@
+// Based on v3
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/title.html
+// https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/title.html
+
+import "./Title.css";
+
+const LinkWrapper = ({ href, children }) => {
+  return href ? (
+    <a className="cmp-title__link" href={href}>
+      {children}
+    </a>
+  ) : (
+    children
+  );
+};
+
+export const Title = ({
+  className = "", // string
+  component = "", // string = "h1" | "h2" | "h3" | "h4" | "h5" | "h6"
+  size = "", // string - "xxl" | "xl" | "lg" | "md" | "sm" | "xs" | "xxs"
```

Comment:
enum might be good for this with a mistype. I would have to go to this component to understand any of these parameters.
Did Christian tell you about the awesome comments? They go like this:
/** Comment here*/ - so when a developer starts to use a component these kind of things like lots of parameters can be described here. Having an interface for the props is nice too so that you can hover over and see what props are needed and typescript can scream at you if something is required.

Reply:
Isn't `enum` a TypeScript thing? I'm not ready to convert these just yet. Looks like I might be able to mimic `enum`s [like this](https://blog.bitsrc.io/explaining-enums-enumerations-in-plain-javascript-895a226622e3#a181), though I'm still not certain how they work.

Reply:
@j2trumpet enum isn't typescript, it's javascript. here's an example:
```
export const TextSize {
 "xl" = "xl",
"lg" = "lg"
}
```
Enums are good at defining stuff where you might want strings but in the case of typescript it helps force the actual string. In reality TextSize.xl = "xl" but the code can just reference the enum instead so the developer doesn't have to care what the string is. The enum string value doesn't have to match either, like I could do
```
export const TextSize {
 "xl" = "extra large yeah I like long strings",
"lg" = "lg"
}
```
Instead, but it makes it so when you are comparing oranges they are truly oranges as well. 🟠  It's a good way to make it if a string happened to need changing, it wouldn't break the code to do that too.